dnsserv_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	. "gopkg.in/check.v1"
+	"net"
+	"testing"
+)
+
+func Test(t *testing.T) { TestingT(t) }
+
+type DNSNameserverSuite struct{}
+
+var _ = Suite(&DNSNameserverSuite{})
+
+func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
+	output := getUDPAddr()
+	host, port, err := net.SplitHostPort(output.String())
+
+	c.Assert(err, IsNil)
+	c.Assert("127.0.0.1", Equals, host)
```

Comment:
"127.0.0.1" and "host" should be reversed. "127.0.0.1" is currently in the "obtained" value while "host" is in "expected"